app/internal/util/util.go

```diff
@@ -80,8 +81,9 @@ func (c *Context) WriteStatusError(err StatusError) {
 func ValidateRequestMethod(ctx *Context, method, err string) error {
```

Comment:
Nit: it wasn't obvious to me that a method called `Validate*` which returned an error would be responsible for writing out the error it produced. When I first read the request handler code before reading the content of this utility, it seemed to me like there was a missing `ctx.WriteStatusError`.

Reply:
Yeah I agree that's confusing, so I just changed the behavior to only return an error and make it the caller's responsibility to write that error.